cf-agent/cf-agent.c

```diff
@@ -2103,13 +2103,13 @@ static int NoteBundleCompliance(const Bundle *bundle, int save_pr_kept, int save
     }
     else
     {
-        Log(LOG_LEVEL_VERBOSE, "A: Promises kept in '%s' = %.0lf", bundle->name, delta_pr_kept);
-        Log(LOG_LEVEL_VERBOSE, "A: Promises not kept in '%s' = %.0lf", bundle->name, delta_pr_notkept);
-        Log(LOG_LEVEL_VERBOSE, "A: Promises repaired in '%s' = %.0lf", bundle->name, delta_pr_repaired);
+        Log(LOG_LEVEL_INFO, "A: Promises kept in '%s' = %.0lf", bundle->name, delta_pr_kept);
+        Log(LOG_LEVEL_INFO, "A: Promises not kept in '%s' = %.0lf", bundle->name, delta_pr_notkept);
```

Comment:
Promises not kept should already be elevated as errors right?

Reply:
Currently there's no way to know which bundles are failing their promises, you have to dig into the source code and infer the promise that failed. This is really useful to avoid hunting across dozens or hundreds of bundles.